src/ceph-volume/ceph_volume/devices/raw/prepare.py

```diff
@@ -90,10 +90,10 @@ def safe_prepare(self, args=None):
     @decorators.needs_root
     def prepare(self):
         secrets = {'cephx_secret': prepare_utils.create_key()}
-        encrypted = 1 if self.args.dmcrypt else 0
-        cephx_lockbox_secret = '' if not encrypted else prepare_utils.create_key()
+        encrypt_devices = self.args.dmcrypt
```

Comment:
I'd suggest a better terminology:

```suggestion
        encrypted_device_types = self.args.dmcrypt
```